eZ Platform/APIs/REST/REST_v3.md

```diff
@@ -0,0 +1,141 @@
+
+## Typical use case
+
+As a front-end developer I would like to:
+* Define a content type in eZ Platform
+* Manage the content via the REST API:
+    * Use the REST endpoints to quickly create a new object
+    * Use the REST endpoints to quickly update an already created object
+    * Use the REST endpoint
+
+# Creating content POST
+The create content endpoint should be simplified to default most values and the simplest possible.
+
+This call should create:
+* A new content object (not a draft) and publish
+* Assign ownership to the current user
+* Create the object in the default language
+* Location can be defined either with ID or URL
+
+## Example: creating a new object
+
+```
+Resource: /content/object
+Method: POST
+{
+    "content-location": "/blogs",
+	"content-type": "blog_post",
+	"fields": {
+		"title": "My blog title",
+		"author": "Bård Farstad",
+		"body": "Body text, ideally simple but also with <b>formatting</b>. Perhaps supporting simple form and docbook.",
+		"image": "/path/to/image.jpg"
+		}
+}
+
+Return:
+HTTP/1.1 201 Created
+Location: /content/object/<newID>
+```
+
+# Fetching content GET
+
+Fetching an object by location. Location can be either a location ID (42) or a relative url (/Blog/My_Post)
+
+## Example: fetching an object
+
+```
+Resource: /content/object/(location_id)
+Method: GET
+
+Return:
+HTTP/1.1 200 Success
+{
+    TODO: add default fields: id, published, modified, version, language etc.
+    "content-location": "/blogs",
+	"content-type": "blog_post",
+    "content-type-name": "Blog Post",
+	"fields": {
+		"title": "My blog title",
+		"author": "Bård Farstad",
+		"body": "Body text, ideally simple but also with <b>formatting</b>. Perhaps supporting simple form and docbook.",
+		"image": "/path/to/image.jpg"
+		}
+}
+```
+# Updating content PUT
+
+The following call updates the complete object.
+
+## Example: updating an object
+
+```
+Resource: /content/object/(location_id)
+Method: PUT
+{
+	"fields": {
+		"title": "My Updated title",
+		"author": "Bård Farstad",
+		"body": "Body text, ideally simple but also with <b>formatting</b>. Perhaps supporting simple form and docbook.",
+		"image": "/path/to/image.jpg"
+		}
+}
+
+Return:
+HTTP/1.1 200 Success
+HTTP/1.1 404 Not found
+
+```
+# Patching content PATCH
+
+The following call patches an object and updates only the defined fields.
+
+## Example: partially updating an object
+
+```
+Resource: /content/object/(location_id)
+Method: PATCH
+{
+	"fields": {
+		"title": "My second update to the title",
+		}
+}
+
+Return:
+HTTP/1.1 200 Success
+HTTP/1.1 404 Not found
+
+```
+# Deleting content DELETE
+
+The following call removes the current content item.
+
+## Example: partially updating an object
+
+```
+Resource: /content/object/(location_id)
+Method: DELETE
+
+
+Return:
+HTTP/1.1 200 Success
```

Comment:
[That's current behaviour](https://github.com/ezsystems/ezpublish-kernel/blob/master/doc/specifications/rest/REST-API-V2.rst#delete-content)

Besides that `200` is [wrong response code here](http://jsonapi.org/format/#crud-deleting)